actors/miner/tests/util.rs

```diff
@@ -2669,48 +2669,10 @@ impl ActorHarness {
     pub fn extend_sectors_versioned(
```

Comment:
I would probably remove this wrapper and just replace all calls to extend_sectors2